gamla/optimized/sync.py

```diff
@@ -377,6 +379,31 @@ def case(*args, **kwargs):
 )
 
 
+_is_iterable = anyjuxt(*map(operator.is_instance)([tuple, list, set, frozenset()]))
```

Comment:
nice, let's change the implementation of `is_iterable` in `functional.py` ?

Reply:
done